rust/tests/loadtest/src/ping.rs

```diff
@@ -120,7 +120,7 @@ pub async fn run_with_cli_args(args: Args) -> anyhow::Result<()> {
         payload_size: args.payload_size,
     };
 
-    let summary = run(config).await?;
+    let summary = run(config, 0).await?;
```

Comment:
I had to pass a dummy seed here. We might expose this to be configurable eventually.